pkg/plugin/aggregation/aggregator.go

```diff
@@ -76,11 +76,11 @@ func NewAggregator(outputDir string, expected []plugin.ExpectedResult) *Aggregat
 }
 
 // Wait blocks until all expected results have come in.
-func (a *Aggregator) Wait(stop chan bool) {
+func (a *Aggregator) Wait(stop chan bool, results chan<- *plugin.Result) {
```

Comment:
I don't love this. Anyone have ideas?

Reply:
I'm kind of confused what this has todo with an annotation change?

Reply:
We need somewhere to tap into the results we're getting so we can update the annotation when we recieve a result.

Reply:
Isn't this very churny on a deamonset plugin, vs. updating a data structure and have periodic batched update?  

e.g. on a 1000 node cluster you'd send 1000 updates.